ui/src/components/search-result.tsx

```diff
@@ -1,149 +1,174 @@
+          <span className="flex flex-row text-[15px] font-medium mb-4 mt-1">
+            {
+              props.type == ResultType.Docment && <img className="inline-block mr-2" src={BlueFolder}></img>
+            }
+            <span className="ml-0 text-[#D5D5D5]">
+              {
+                props.type == ResultType.Message && <span>#</span>
+              }
+              {props.location} ·&thinsp;
+            </span>
+            <span className="flex flex-row items-center">
+              <img className="inline-block ml-2 mr-2 h-4 rounded-xl" src={props.author_image_data ? props.author_image_data : props.author_image_url}></img>
+              <span className='capitalize'>{props.author} ·</span>
+            </span>
+            <span>
+              &thinsp;Updated {getFormattedTime(props.time)}&thinsp; |&thinsp;
+            </span>
+            <span className="flex flex-row items-center">
+              {getSmallIconByPlatform(props.platform as Platform)}
+              <span className="text-[#A3A3A3]">{props.platform}</span>
+            </span>
+          </span>
+
+          {props.type == ResultType.Docment &&
+            <span>
+              {props.content.map((text_part, index) => {
+                return (
+                  <span key={index} style={{ wordBreak: 'break-word' }} className={(text_part.bold ? 'font-bold text-white' : '') +
+                    " text-md font-poppins font-medium"}>
+                    {text_part.content}
+                  </span>
+                )
+              })}
+            </span>
+          }
+          {props.type == ResultType.Message &&
+            <p className="bg-[#352C45] p-2 px-4 rounded-lg font-poppins leading-[28px] border-b-[#916CCD] border-b-2">
+              {props.content.map((text_part, index) => {
+                return (
+                  <span key={index} className={(text_part.bold ? 'font-bold text-white' : '') +
+                    " fony-[14px] font-regular"}>
+                    {text_part.content}
+                  </span>
+                )
+              })}
+            </p>
+          }
+        </p>
       </div>
-    );
+    </div>
+  );
 }
 
 
-function getFormattedTime (time: string) {
-    let date = new Date(time);
-    return date.toLocaleDateString('en-US', {month: 'short', day: 'numeric', year: 'numeric'});
+function getFormattedTime(time: string) {
+  let date = new Date(time);
+  return date.toLocaleDateString('en-US', { month: 'short', day: 'numeric', year: 'numeric' });
 }
 
 function getSmallIconByPlatform(platform: Platform) {
-    let classes = "inline mx-1 fill-[#A3A3A3]";
-    switch (platform) {
-      case Platform.Confluence:
-        return <FaConfluence className={classes}></FaConfluence>
-      case Platform.Slack:
-        return <FaSlack className={classes}></FaSlack>
-      case Platform.Drive:
-        return <FaGoogleDrive className={classes}></FaGoogleDrive>
-    }
+  let classes = "inline mx-1 fill-[#A3A3A3]";
```

Comment:
No need to touch function's indentation, it shows as code change that isn't relevant to this issue.